support/ipfs-run.sh

```diff
@@ -1,13 +1,17 @@
 #!/bin/bash
 
-ipfs daemon &
-sleep 5s
+if [ "$RUN_LOCAL_ONLY" != "true" ]; then
+  ipfs daemon &
+  sleep 10s
```

Comment:
Minor improvement — you could listen for the status of the ipfs process + pause the script.

Reply:
I went to easy route and put in a sleep. I agree could have put in a do loop to process based on standard out.